### PR TITLE
Clarify directory_mode parameter usage-#81127

### PR DESCRIPTION
**Pull request by rakeshkhopade12**
##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by rakeshkhopade12** [comment]
fixed#81127

File: lib/ansible/modules/copy.py [hunk 1]
```diff
@@ -82,6 +82,7 @@
     - When doing a recursive copy set the mode for the directories.
     - If this is not set we will use the system defaults.
     - The mode is only set on directories which are newly created, and will not affect those that already existed.
+    - parameter requires a numeric permissions mode in octal format.
```
Comment (mkrizek) [inline lib/ansible/modules/copy.py:85]: The value should be quoted to work properly.

**Comment by mkrizek** [comment]
We are going to close this in favor of https://github.com/ansible/ansible/pull/81127 where the review is in progress.

If you have further questions please stop by IRC or the mailing list:
* IRC: #ansible on irc.libera.chat
* mailing list: https://groups.google.com/forum/#!forum/ansible-project

File: lib/ansible/modules/copy.py (hunk 1 above)
Comment (rakeshkhopade12) [reply lib/ansible/modules/copy.py:85]: Thank you for your attention and changes

**Pull request closed without merging**